public/ACMW/css/events.css

```diff
@@ -42,16 +33,16 @@
 	margin-bottom: 0;
 	background-color: #2d2d30;
 	border: 0;
-	font-size: 11px;
+	font-size: 11px ;
 	letter-spacing: 4px;
 	opacity: 0.9;
 }
-.navbar li a,
+.navbar li a, 
 .navbar .navbar-brand {
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)